.github/workflows/publish.yml

```diff
@@ -12,7 +12,7 @@ on:
 jobs:
   publish:
     name: publish
-    runs-on: ubuntu-16.04
+    runs-on: macos-latest
```

Comment:
Could we keep running this on `ubuntu`? I prefer `ubuntu` as the "default" OS, as that is easier to debug for me (and, in general, somewhat more sane than windows & mac for these sorts of automated things).

Reply:
Done. My dev env is mac, and I wasn't sure how differently `sed` behaves. But I tested this change now and moved this to ubuntu.